docs/paper/dev/guides/pdc.md

```diff
@@ -0,0 +1,156 @@
+---
+slug: /dev/pdc
+---
+
+# Persistent Data Container (PDC)
+
+The Persistent Data Container (PDC) is a way to store custom data on a whole range of objects; such as items, entities, and blocks.
+The Full list of classes that support the PDC are:
+
+- [Chunk](#chunk)
+- [World](#world)
+- [Entity](#entity)
+- [TileState](#tilestate)
+- [ItemMeta](#itemmeta)
+- [Structure](#structure)
+
+## What is it used for?
+In the past, developers resorted to a variety of methods to store custom data on objects:
+
+- NBT tags: Requires reflection to access internals and was generally unreliable in the long term.
+- Lore and display names: Prone to collisions as well as slow to access.
+
+The benefit of the PDC is that it allows for a more reliable and performant way to store arbitrary data on objects.
+It also doesn't rely on accessing server internals, so it's less likely to break on future versions. It also removes the need to 
+manually track the data lifecycle, as, for example with an entity, the PDC will be saved when the entity unloads.
+
+## Adding Data
+To store data in the PDC, there are a few things you need first. The first is a `NamespacedKey` which is used to identify the data.
+The second is a `PersistentDataContainer` which is the object you want to store the data on. The third is the data itself.
+
+```java
+// Create a NamespacedKey
+NamespacedKey key = new NamespacedKey(pluginInstance, "example-key");
+
+ItemStack item = new ItemStack(Material.DIAMOND);
+// ItemMeta implements PersistentDataHolder so we can get the PDC from it
+ItemMeta meta = item.getItemMeta();
+meta.getPersistentDataContainer().set(key, PersistentDataType.STRING, "I love Tacos!");
+item.setItemMeta(meta);
+```
+
+:::info
+
+It is considered good practice to reuse the `NamespacedKey` objects. They can be constructed with either:
+- A `Plugin` instance and a `String` identifier
+- A `String` namespace and a `String` identifier
+
+The first option is often preferred as it will automatically use the plugin's namespace, however the second option can be used if you
+want to use a different namespace / access the data from another plugin.
+
+:::
+
+## Getting Data
+To get data from the PDC, you need to know the `NamespacedKey` and the `PersistentDataType` of the data.
+
+```java
+// Create a NamespacedKey
+NamespacedKey key = new NamespacedKey(pluginInstance, "example-key");
+
+ItemStack item = ...; // Retrieve the item from before
+// Get the data from the PDC
+PersistentDataContainer container = item.getItemMeta().getPersistentDataContainer();
+if (container.has(key, PersistentDataType.STRING)) {
+    String value = container.get(key, PersistentDataType.STRING);
+    // Do something with the value
+    player.sendMessage(Compontent.text(value));
```

Comment:
Small typo Compontent instead of Component

Reply:
Thanks :)